packages/core/test/files/counter_in_running.html

```diff
@@ -0,0 +1,338 @@
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+        </p>
+        <p>
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+        </p>
+        <p>
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+          placeholder placeholder placeholder placeholder placeholder
+        </p>
+
+        <div class="reset-page">reset page</div>
+
```

Comment:
This `<div class="reset-page">reset page</div>` element is not used for resetting page counter. So I think this should be removed.



```suggestion
```

Reply:
ok